tox.ini

```diff
@@ -24,10 +24,14 @@ deps =
 extras =
     azure
     docker
+    docs
+    ec2
     gce
+    linode
     lxc
     openstack
     vagrant
+    windows
```

Comment:
Is this conditional in `setup.cfg`

Reply:
I don't understand ?

Reply:
I'm curious whether it will try installing incompatible dep under Linux and then running tests against it

Reply:
Ah ! certainly not, the windows extra is to install pywinrm, the module to be able to connect to windows from ansible.